color/src/dynamic.rs

```diff
@@ -365,6 +369,16 @@ impl DynamicColor {
     }
 }
 
+impl Default for DynamicColor {
+    fn default() -> Self {
+        Self {
```

Comment:
I think this would read more smoothly as `Self::from_alpha_color(AlphaColor::<Srgb>::default())` (or WHITE if we wanted to be explicit, I'm a bit torn).

Reply:
Done.